docs/dev/install.rst

```diff
@@ -194,3 +194,70 @@ If you ever need to bypass Therapist, you can do so by passing
 ``--no-verify`` to your ``git commit`` command.
 
 .. _Therapist: http://therapist.readthedocs.io/en/latest/overview.html
+
+.. _remotesettings-install:
+
+Remote Settings
+---------------
+If you want to enable the publication of recipes on `Remote Settings`_, you'll need to set up
+an instance locally.
+
+1. Follow the `Remote Settings installation instructions`_ to launch a development
+   instance of Remote Settings locally.
+
+2. Configure Normandy to enable the integration by adding the following lines
+   to ``.env`` (create the file if it does not exist yet):
+
+   .. code-block:: ini
+
+      DJANGO_REMOTE_SETTINGS_URL=http://localhost:8888/v1
+      DJANGO_REMOTE_SETTINGS_USERNAME=normandy
+      DJANGO_REMOTE_SETTINGS_PASSWORD=n0rm4ndy
+
+3. Adjust the Remote Settings configuration in ``server.ini`` for the Normandy
+   recipes collection, and restart the Remote Settings container:
+
+   .. code-block:: ini
+
+      kinto.signer.main-workspace.normandy-recipes.to_review_enabled = false
+      kinto.signer.main-workspace.normandy-recipes.group_check_enabled = false
+
+4. Create the dedicated Remote Settings user and collection for Normandy:
+
+   .. code-block:: bash
+
+      SERVER=http://localhost:8888/v1
+
+      curl -X PUT ${SERVER}/accounts/normandy \
+           -d '{"data": {"password": "n0rm4ndy"}}' \
+           -H 'Content-Type:application/json'
+
+      curl -X PUT ${SERVER}/buckets/main-workspace/collections/normandy-recipes \
+           -H 'Content-Type:application/json' \
+           -u 'normandy:n0rm4ndy'
```

Comment:
These failed for me. Output:

```
{"permissions":{"write":["account:normandy"]},"data":{"password":"$2b$12...notsureitmatters...Zff5b4Mcem","id":"normandy","last_modified":1543510196665}}

{"code":403,"errno":121,"error":"Forbidden","message":"This user cannot access this resource."}
```

I followed the instructions above. I added the two `kinto.signer...` lines under the `[app:main]` section and made sure I stopped and started the server.

Reply:
Looks like autograph can't be reached

Reply:
:)

I think the pressing issue is that the (normandy) instructions don't explain how to set up autograph. Perhaps it's because normandy already has some and I'm just failing that independent of remote-settings :)

Reply:
Normandy's docs for setting up Autograph are here: https://normandy.readthedocs.io/en/latest/dev/install.html#autograph. You'll need to configure remote-settings to talk to Autograph as well.